### ISSUE TITLE
You can /gcmd other admins

### ISSUE THREAD

**Issue by LegendIsAwesomes** [TotalFreedom/TotalFreedomMod#1495]
Why not use an if statement checking if they're admin, if they are get a message saying 'Don't GCMD admins' or something.

**Comment by RobinGall2910** [comment]
no pls

**Comment by DUFCLiam** [comment]
Why? @RobinGall2910

**Comment by RobinGall2910** [comment]
@DUFCLiam can this just be untouched as is, it's better like that

**Comment by xItsNickHD** [comment]
It's because @RobinGall2910 has fun abusing it C::

**Comment by LegendIsAwesomes** [comment]
Tru

**Comment by PieGuy7896** [comment]
Admins should be smart enough in the first place not to do that @LegendIsAwesomes

**Comment by LegendIsAwesomes** [comment]
@PieGuy7896 I know but they still do it.

**Comment by OxLemonxO** [comment]
why the hell is pie here
anyway, just keep it the way it is.

**Comment by ShadowArmadyl** [comment]
As @PieGuy7896 said above. The reason we have admins is because they can be trusted not to abuse and uphold the rules. Removing all opportunity of possible abusing will make the server no longer "TotalFreedom".

**Comment by OxVeganMasterxO** [comment]
Wat @PieGuy7896 is here .-. Pie is on TF GitHub :O

**Comment by Wild1145** [comment]
I would be inclined to partially agree. While I don't think you should be able to /gcmd every admin, you should be able to use it on any rank lower than your own, so a senior could do it to a telnet, but not the other way around. I feel that would be the smartest solution to implement here.

**Comment by JeromSar** [comment]
I believe this command is sometimes put to good use, and therefore shouldn't be blocked. One important aspect of adminship is the trust invested by the server manager into the admins. If admins do misuse this, they are not fit for admin. On the of chance that this command is necessary, I think it should be available.

**Comment by Wild1145** [comment]
@JeromSar I just cannot see a reason you should ever need to gcmd someone of a higher rank? That's why my suggestion of it being permission based so you can only gcmd someone of an equal or lower rank? That's what we did for a lot of the IcarusMod commands like this and it worked well.

**Comment by JeromSar** [comment]
@Wild1145 Okay, perhaps you're right. As long as you can at least /gcmd someone of equal rank.

**Comment by Wild1145** [comment]
@JeromSar Yeah, that's how we did it for IcarusMod, I just cant see a need for a super to gcmd a senior without it basically being abusive
